runtime/src/main/java/org/corfudb/runtime/view/metadata/IMetadataServiceView.java

```diff
@@ -0,0 +1,40 @@
+package org.corfudb.runtime.view.metadata;
+
+import java.util.UUID;
+
+/**
+ * This interface defines the operations provided by the metedata service.
+ */
+
+public interface IMetadataServiceView {
+
+    /**
+     * Create a stream
+     * @param stream The stream to create
+     * @throws StreamAlreadyExistsException This exception is thrown if a previous create opertaion
+     * succeeded with the same stream uuid
+     */
+    void createStream(UUID stream) throws StreamAlreadyExistsException;
```

Comment:
What if you acquire a token but dont write to it? Since an operation that spans a token acquisition and metadata storeupdate is not atomic, then I don't feel like having a stream start address makes sense because it might not be accurate. Maybe just pass the current global tail address instead of passing the first stream address? Using the global tail when creating the stream would tell us no entry belonging to this stream would exist before that tail (i.e. hard lower watermark when traversing the backpointers)